docs/content/usage/configuration/credentials.md

```diff
@@ -5,6 +5,13 @@ draft: false
 description: "Configure credentials that the Infection Monkey will use for propagation."
 ---
 
-On this screen you can feed the Infection Monkey "stolen" credentials from your network, simulating an attacker with inside knowledge.
+On this screen, you can configure credentials for Infection Monkey to use in the propagation
+stage. These can be anything from default or weak passwords, to "stolen" credentials from your
+network, simulating an attacker with inside knowledge.
+
+All possible credential combinations will be tried for propagation. Identity-secret pairs added
```

Comment:
We may want to define "identity" and "secret". It seems obvious, but I have a feeling it may be valuable for us to explicitly list what kinds of identities and what kinds of secrets we accept.